packages/csharp/PortingAssistant/PortingAssistant.Common/Services/AssesmentService.cs

```diff
@@ -36,6 +37,23 @@ public class AssessmentService : IAssessmentService
                 var solutionAnalysisResult = _client.AnalyzeSolutionAsync(request.solutionFilePath, request.settings);
                 solutionAnalysisResult.Wait();
 
+                _logger.LogInformation("get preTriggerData");
+                var preProjectTriggerDataDictionary = new Dictionary<string, ProjectTableData>();
+                if (request.preTriggerData != null && request.preTriggerData.Length > 0)
+                {
+                    Array.ForEach(request.preTriggerData, prop => {
+                        var proj = JsonConvert.DeserializeObject<ProjectTableData>(prop);
+                        _logger.LogInformation(proj.projectName + ": incompatibleApi " + proj.incompatibleApis);
+                        if (!preProjectTriggerDataDictionary.ContainsKey(proj.projectName))
+                        {
+                            preProjectTriggerDataDictionary.Add(proj.projectName, proj);
+                        }
+                    }
+                    );
+                    _logger.LogInformation("request.preTriggerData.keys:" + String.Join("::", preProjectTriggerDataDictionary.Keys));
```

Comment:
can we remove the debug logging statements?

Reply:
yes, i removed